lib/metadataTypes/Asset.js

```diff
@@ -495,6 +495,78 @@ class Asset extends MetadataType {
         return metadata;
     }
 
+    /**
+     * Gets executed after deployment of metadata type
+     *
+     * @param {TYPE.MetadataTypeMap} metadata metadata mapped by their keyField
+     * @param {TYPE.MetadataTypeMap} _ originalMetadata to be updated (contains additioanl fields)
+     * @param {{created: number, updated: number}} createdUpdated counter representing successful creates/updates
+     * @param {boolean} [isRefresh] optional flag to indicate that triggeredSend should be refreshed after deployment of assets
+     * @returns {Promise.<void>} -
+     */
+    static async postDeployTasks(metadata, _, createdUpdated, isRefresh) {
```

Comment:
things like this flags, ideally, shouldnt be passed, but be available via a context object to avoid that we pass 1000 things via method

Reply:
I do agree when it comes to global flags. But this one is limited to this command and hence I don't see the point to clutter a central class, e.g. util, with this variable.
This is a parameter that is used specifically for this case and hence it should continue to be a function parameter of this function

Reply:
your comment is a little contradictory. You want to add this to all common classes because it is needed here? having a context class is exactly the kind of thing that should go via a context class to ensure you DONT have to pollute all the other methods with additional parameters. generally, beyond 3 parameters suggests something else is not well designed

Reply:
We will actually have a second use case: attribute groups on child bus when deploying shared data extensions.

Currently working on getting these fully resolved on another branch